src/holofuel/pages/CreateOfferRequest/CreateOfferRequest.js

```diff
@@ -87,10 +101,16 @@ export default function CreateOfferRequest ({ history: { push } }) {
   const [total, setTotal] = useState(0)
 
   const onAmountChange = amount => {
-    if (isNaN(amount)) return
-    const newFee = Number(amount) * FEE_PERCENTAGE
-    setFee(newFee)
-    setTotal(Number(amount) + newFee)
+    if (isNaN(amount)) return null
```

Comment:
don't need to explicitly return `null` here and it's slightly confusing. Makes it seem like we care about the return value when actually we're just exiting early.

Reply:
This is purely to abide by the linting rules. I agree that it's unnecessary otherwise.

Reply:
there's no linting rule that specifies you have to return a value.

`if (isNaN(amount)) return` should make the linter happy